packages/firestore/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 # Unreleased
 - [changed] Firestore now recovers more quickly after network connectivity
   changes (airplane mode, Wi-Fi availability, etc.).
+- [feature] Added `clearPersistence()`, clears the persistent storage
```

Comment:
Don't use commas in place of words like "which" and "that". Also "cached tests" should be "cached documents".

I think we can drop the "For applications ..." bit here.

This should refer to any issues we're addressing (in this case, 449).